.storybook/decorators.tsx

```diff
@@ -7,25 +7,25 @@ interface Themes {
 
 const themesData: { [key: string]: Themes } = {
   'corporate-light': {
-    cssPath: '/node_modules/@boston-scientific/anatomy-tokens/lib/css/corporate/light.css',
+    cssPath: 'https://cdn.jsdelivr.net/npm/@boston-scientific/anatomy-tokens@5.0.0-beta.15/lib/css/corporate/light.css',
```

Comment:
do we need to update these to .16 now? is there a way to automate this like we do in the docs site?